fitbenchmarking/results_processing/local_min_table.py

```diff
@@ -198,3 +198,4 @@ def save_colourbar(self, fig_dir, n_divs=2, sz_in=None) -> str:
         if sz_in is not None:
             return super().save_colourbar(fig_dir, n_divs=2, sz_in=sz_in)
         return super().save_colourbar(fig_dir, n_divs=2)
+    # pylint: enable=unused-argument
```

Comment:
Do these tags need to be re enabled? I thought they were scoped meaning this would be enabled after the function exited?